README.md

```diff
@@ -53,6 +53,44 @@ Remove the following line in the mentioned file: it's not needed anymore:
 - import '@symfony/autoimport';
 ```
 
+## The controllers.json File
+
+The bridge works by reading a `controllers.json` file that is automatically
+updated by Symfony Flex whenever you download a UX-powered package. For
+example, after running `composer require symfony/ux-dropzone.js`, it will
+look like this:
+
+```json
+{
+    "controllers": {
+        "@symfony/ux-dropzone": {
+            "dropzone": {
+                "enabled": true,
+                "webpackMode": "eager",
+                "autoimport": {
+                    "@symfony/ux-dropzone/src/style.css": true
+                }
+            }
+        }
+    },
+    "entrypoints": []
+}
+```
+
+Each item under `controllers` will cause a Stimulus controller to be
+registered with a specific name - in this case the controller would
+be called `symfony--ux-dropzone--dropzone` (the `/` becomes `--`).
+
+By default, the new controller will be always be included in your
+JavaScript package. You can control that with the `webpackMode` option,
+ordered from least to most lazy:
+
+* `webpackMode: 'eager'`: include in JavaScript that is downloaded on page load
+* `webpackMode: 'lazy'`: isolate into a separate JavaScript file, but immediately
+  download this after the page is loaded.
+* `webpackMode: 'controller-lazy'`: wait until this controller HTML is
```

Comment:
This wording feels incorrect, but I'm not sure how to improve it :) . 

`wait until this controller is used in a page and *then* ...`?